roles/usegalaxy_eu.galaxy_cleanup/tasks/main.yml

```diff
@@ -0,0 +1,32 @@
+---
+- name: Verify galaxy_cleanup_scratchstorage configuration
+  block:
+    - name: Check if galaxy_cleanup_scratchstorage is not empty
+      assert:
+        that:
+          - galaxy_cleanup_scratchstorage | length > 0
+        fail_msg: "galaxy_cleanup_scratchstorage is empty"
+        success_msg: "galaxy_cleanup_scratchstorage is configured"
+
+    - name: Check if objectstore_id includes 'scratch'
+      assert:
+        that:
+          - galaxy_cleanup_scratchstorage | selectattr('objectstore_id', 'search', 'scratch') | list | length > 0
+        fail_msg: "No objectstore_id in galaxy_cleanup_scratchstorage contains 'scratch'"
+        success_msg: "At least one objectstore_id contains 'scratch'"
+
+- name: Install cleanup scratch script
+  template:
+    src: "clean_scratch_storage.sh.j2"
+    dest: "/usr/local/bin/clean_scratch_storage.sh"
+    mode: "0555"
+  when: galaxy_cleanup_scratchstorage | length > 0
+
+- name: Schedule Galaxy cleanup objectstore script
+  ansible.builtin.cron:
+    name: "Galaxy cleanup scratch"
+    job: "/usr/local/bin/clean_scratch_storage.sh"
+    minute: "{{ galaxy_cleanup_minute }}"
+    hour: "{{ galaxy_cleanup_hour }}"
+    user: "{{ galaxy_user.name }}"
+  when: galaxy_cleanup_scratchstorage | length > 0
```

Comment:
```suggestion
```
Same here, if ansible fails in a previous step, the steps below will not be executed.